src/main/java/martin/Ui.java

```diff
@@ -0,0 +1,37 @@
+package martin;
+import java.util.Scanner;
+
```

Comment:
Consider adding JavaDocs for classes for a short description of what the class encapsulates or can do.